### DISCUSSION TITLE
Running total in accounts: how to turn it back on?

### DISCUSSION THREAD

**Discussion by MiriamNZ** [actualbudget/releases#222] [Q&A]
My accounts no longer have a running total.
I can't find a setting to turn this back on. Actual App on MacOS.
Can anyone point me to it?
Cheers
Miriam

**Comment by Silvenga** [comment]
Not here? (set per account)

![image](https://user-images.githubusercontent.com/2753330/156081428-9611de4b-9671-40e3-b879-c0418597940e.png)

**Reply by Deguello** [reply]
Yeah. Three dot menu just above the Balance column.

**Reply by Rick-Thomas** [reply]
Do you have to set it for each account?   Or is there a option to set RUNNING BALANCE for ALL accounts?

**Reply by Silvenga** [reply]
I've only seen it per account @laburante.
